src/ochanticipy/config/countryconfig.py

```diff
@@ -7,37 +7,61 @@
 from ochanticipy.utils.io import parse_yaml
 
 
+class CustomLayerNames(BaseModel):
```

Comment:
Can remove this class, that way you won't need "name" in front of the custom layer names which is redundant